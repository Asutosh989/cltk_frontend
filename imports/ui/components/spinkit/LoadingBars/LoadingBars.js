import React from 'react';

export default class LoadingBars extends React.Component {
	render() {
		return (
			<div className="loading-spinner loading-spinner-bars">
				<div className="rect1"></div>
				<div className="rect2"></div>
				<div className="rect3"></div>
				<div className="rect4"></div>
				<div className="rect5"></div>
			</div>
		);
	}
}
