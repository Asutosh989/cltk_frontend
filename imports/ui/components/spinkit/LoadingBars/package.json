{
  "name": "LoadingBars",
  "version": "0.0.1",
  "private": true,
  "main": "./LoadingBars.js"
}
