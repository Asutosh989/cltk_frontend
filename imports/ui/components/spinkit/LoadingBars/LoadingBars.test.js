
import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

// component:
import LoadingBars from './LoadingBars';

describe('LoadingBars', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(
				<MuiThemeProvider>
					<LoadingBars />
				</MuiThemeProvider>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
