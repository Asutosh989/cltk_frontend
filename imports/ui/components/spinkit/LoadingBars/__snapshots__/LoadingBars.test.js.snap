// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingBars renders correctly 1`] = `
<div
  className="loading-spinner loading-spinner-bars"
>
  <div
    className="rect1"
  />
  <div
    className="rect2"
  />
  <div
    className="rect3"
  />
  <div
    className="rect4"
  />
  <div
    className="rect5"
  />
</div>
`;
